<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainTabWidget</class>
 <widget class="QWidget" name="mainTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>331</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabWidgetPage1">
    <attribute name="title">
     <string>Производительность</string>
    </attribute>
    <widget class="perfomanceWidget" name="perfWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage2">
    <attribute name="title">
     <string>Управление</string>
    </attribute>
    <widget class="logicWidget" name="managementWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>661</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="consoleTextWidget" name="consoleTextEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>671</width>
     <height>101</height>
    </rect>
   </property>
   <property name="documentTitle">
    <string>Console</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>perfomanceWidget</class>
   <extends>QWidget</extends>
   <header>widgets/perfomancewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>consoleTextWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>widgets/consoletextwidget.h</header>
  </customwidget>
  <customwidget>
   <class>logicWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/logicwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
